infrastructure/cdn-in-a-box/traffic_ops/adduser.pl

```diff
@@ -24,23 +24,27 @@
 use Crypt::ScryptKDF qw{ scrypt_hash };
 
 if ($#ARGV < 2) {
-    die "Usage: $ARGV[0] <username> <password> <role>\n";
+    die "Usage: $ARGV[0] USERNAME PASSWORD [ROLE] [TENANT] [EMAIL] [FULL_NAME]\n";
 }
 
 my $username = shift // 'admin';
 my $password = shift or die "Password is required\n";
 my $role = shift // 'admin';
 my $tenant = shift // 'root';
+my $email = shift // 'admin@no-reply.trafficops.infra.ciab.test';
+my $full_name = shift // 'James Cole'
```

Comment:
`James Cole` should be an environment variable, Perl can be used to retrieve it.

Reply:
It could be, but that's following the pattern of allowing the script to add a generic user to the database while defaulting to the "admin" user's default information.

What about:
```perl
my $full_name = shift // $ENV{TO_ADMIN_FULL_NAME};
```
? And I'd wanna do the same thing with the email.

Reply:
Yeah that looks good.